OpenEdXMobile/src/main/java/org/edx/mobile/user/SaveUriToFileTask.java

```diff
@@ -63,10 +63,10 @@ private static Uri getFileUriFromMediaStoreUri(@NonNull Context context, @NonNul
             return null;
         }
         try {
-            if (cursor.getCount() != 1) {
+            cursor.moveToFirst();
```

Comment:
ok @farhan  but i nedd this check cursor.getColoumnCount() > 0 because some times cursor not null but coloumn count is 0 and app crash on getString(0)